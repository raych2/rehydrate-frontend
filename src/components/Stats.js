const Stats = () => {
  return (
    <>
      <div className="topicSection smallerTopicSection">
        <div className="mainHeader">
          How to Access <div className="rectangle shorterTitle">Stats</div>?
        </div>
        <div className="infoContainer">
          <div className="subHeader">Annual Reports</div>
          <div className="subText">
            Corporations are obligations to publish annual reports. Because of
            the climate crisis and the rising trend of "sustainability," the
            reports have increasingly included environmental policies,
            practices, and charities.
          </div>
        </div>
        <div className="infoContainer">
          <div className="subHeader">Sustainable Practices</div>
          <div className="subText">
            Corporations have published non-financial reports to state their
            environmental and social performances. If some companies do not have
            these, then we will dig into their financial reports on water usage
            tied to the workplace and suppliers.
          </div>
          <div className="subText">
            Here are some key words: "community service", "social impact",
            "water management", "water usage"
          </div>
        </div>
      </div>
    </>
  );
};

export default Stats;
