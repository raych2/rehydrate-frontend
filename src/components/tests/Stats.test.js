import { render } from '@testing-library/react';
import Stats from '../Stats';

describe('Stats component', () => {
  test('renders correctly', () => {
    const { container } = render(<Stats />);
    expect(container).toMatchSnapshot();
  });
});
