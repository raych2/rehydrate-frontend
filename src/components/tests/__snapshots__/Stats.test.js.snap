// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats component renders correctly 1`] = `
<div>
  <div
    class="topicSection smallerTopicSection"
  >
    <div
      class="mainHeader"
    >
      How to Access 
      <div
        class="rectangle shorterTitle"
      >
        Stats
      </div>
      ?
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Annual Reports
      </div>
      <div
        class="subText"
      >
        Corporations are obligations to publish annual reports. Because of the climate crisis and the rising trend of "sustainability," the reports have increasingly included environmental policies, practices, and charities.
      </div>
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Sustainable Practices
      </div>
      <div
        class="subText"
      >
        Corporations have published non-financial reports to state their environmental and social performances. If some companies do not have these, then we will dig into their financial reports on water usage tied to the workplace and suppliers.
      </div>
      <div
        class="subText"
      >
        Here are some key words: "community service", "social impact", "water management", "water usage"
      </div>
    </div>
  </div>
</div>
`;
